src/js/constants/JobStates.ts

```diff
@@ -2,7 +2,16 @@
  * Sort order is ordered by most important (lowest number, top of list)
  * to least important (largest number, bottom of list)
  */
-const JobStates = {
+
+export interface IJobStates {
```

Comment:
Back to the discussion about the "I". it looks a little mixed to me that we use some places and others not.

Reply:
I guess there is a good reason for that, but we should be consistent so either do it or not.

Reply:
This should not be called IJobStates after our conversation on dropping the I.

Reply:
Fixed that 👍